apps/server/prisma/schema.prisma

```diff
@@ -57,3 +58,18 @@ model Ingredients {
   name      String
   imageName String
 }
+
+model contributionId {
+  id        String        @id @default(uuid())
+  contributor String
+  type      ContributionType
+  dateTime  DateTime
+  value     Float
+  createAt  DateTime      @default(now())
```

Comment:
`create` is a Verb. We can name it like `createdAt`

Reply:
it is done